tests/test_client.py

```diff
@@ -773,6 +773,13 @@ def create_refunds(self, start_date, num_records, payment_response=None):
             refunds += created_refund
         return refunds
 
+    @backoff.on_exception(
+        backoff.expo,
+        requests.exceptions.RequestException,
```

Comment:
This needs to include the RuntimeError from line 821 or make line 821 raise a RequestException.